.eslintrc.json

```diff
@@ -15,6 +15,7 @@
     "jsx-a11y/anchor-is-valid": [ "error", {
       "aspects": [ "noHref" ]
     }],
+    "jsx-a11y/label-has-for": "Off",
```

Comment:
I think this should be `["off"]`

Reply:
If Off is in danger of being parsed as and string and the in turn cast to  a boolean true,  do you mind if I change it to 0 [zero]?